children/schema.py

```diff
@@ -96,3 +102,12 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
         return SubmitChildMutation(
             child=child, relationship=relationship, guardian=guardian
         )
+
+
+class Query:
+    children = relay.ConnectionField(ChildConnection)
```

Comment:
you can also do:
```
from graphene_django import DjangoConnectionField

...

class Query:
    children = DjangoConnectionField(ChildNode)

    ...

```
this way you don't even hav to declare `ChildConnection`, as graphene will generate it for you.

Reply:
having an explicit `Connection` class is called for, if you need to add some of your own sugar to it, like Pekka did in [profile](https://github.com/City-of-Helsinki/open-city-profile/blob/develop/profiles/schema.py#L59)

Reply:
That is actually a bit odd IMO. For normal `ConnectionField` you need an explicit `Connection` now (they seem to have changed it to work like that), but for `DjangoConnectionField` you don't.

But yeah I'll change those, good point.

Reply:
yeah, it was a change in Graphene 2.0. don't know why it only affect regular ConnectionFields.

Reply:
Changed.